lib/modules/manager/bitbucket-pipelines/extract.ts

```diff
@@ -59,3 +38,43 @@ export function extractPackageFile(content: string): PackageFileContent | null {
   }
   return { deps };
 }
+function addDepAsBitbucketTag(
+  deps: PackageDependency<Record<string, any>>[],
+  pipe: string
+): void {
+  const [depName, currentValue] = pipe.split(':');
+
+  const dep: PackageDependency = {
+    depName,
+    currentValue,
+    datasource: 'bitbucket-tags',
+  };
+
+  logger.trace(
+    {
+      depName: dep.depName,
+      currentValue: dep.currentValue,
+    },
+    'Bitbucket pipe tag'
+  );
+  dep.depType = 'bitbucket-tags';
```

Comment:
Why do you set the depType to the same values as datasources?
`depType` is used to provide convenience grouping for users. See the [terraform manager](https://github.com/renovatebot/renovate/tree/main/lib/modules/manager/terraform#disabling-parts-of-the-manager) for example.

Reply:
This is existing logic which I didn't want to change

Reply:
I can make this unrelated change if you like

Reply:
K, in that case it is fine. 

But IMO we should remove this in a major as this is a breaking change to be more consistent.

Reply:
It's been removed @secustor